File: demos/odd-even.java
public class EvenOdd {
    public static void main(String[] args) {

        int n = 7;
        int rem = n % 2;

        if (rem == 0) {
            System.out.println(n + " is even");
        } else {
            System.out.println(n + " is odd");
        }

    }
}

File: demos/odd-even.c
#include <stdio.h>

int main() {

    int n = 7;
    int rem = n % 2;

    if (rem == 0) {
        printf("%d is even\n", n);
    } else {
        printf("%d is odd\n", n);
    }

    return 0;
}